tests/expectations/metrics/test_core.py

```diff
@@ -612,6 +615,611 @@ def test_column_histogram_metric_spark(spark_session):
     assert results == {desired_metric.id: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1]}
 
 
+def test_column_partition_metric_pd():
+    week_idx: int
```

Comment:
In some tests we have a "what does this test and why?".. would we be able to add a doc string? I'd hate to lose out on preserving the thoughtwork that went into this.